engine/src/main/java/org/terasology/engine/rendering/assets/skeletalmesh/SkeletalMeshDataBuilder.java

```diff
@@ -32,18 +32,18 @@ public SkeletalMeshDataBuilder addBone(Bone bone) {
         return this;
     }
 
-    public SkeletalMeshDataBuilder addVertices(List<Vector3f> vertices) {
-        this.vertices.addAll(vertices);
+    public SkeletalMeshDataBuilder addVertices(List<Vector3f> verticesToAdd) {
```

Comment:
I think these renames are unnecessary and the code would be better readable if we leave it as is...